hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ContainerKeyMapperTask.java

```diff
@@ -156,19 +178,70 @@ public Pair<String, Boolean> process(OMUpdateEventBatch events) {
         return new ImmutablePair<>(getTaskName(), false);
       }
     }
+    try {
+      writeToTheDB(containerKeyMap, containerKeyCountMap, deletedKeyCountList);
+    } catch (IOException e) {
+      LOG.error("Unable to write Container Key Prefix data in Recon DB.", e);
+      return new ImmutablePair<>(getTaskName(), false);
+    }
     LOG.info("{} successfully processed {} OM DB update event(s).",
         getTaskName(), eventCount);
     return new ImmutablePair<>(getTaskName(), true);
   }
 
+  private void writeToTheDB(Map<ContainerKeyPrefix, Integer> containerKeyMap,
+                            Map<Long, Long> containerKeyCountMap,
+                            List<ContainerKeyPrefix> deletedContainerKeyList)
+      throws IOException {
+    RDBBatchOperation rdbBatchOperation = new RDBBatchOperation();
+    containerKeyMap.keySet().forEach((ContainerKeyPrefix key) -> {
+      try {
+        reconContainerMetadataManager
+            .batchStoreContainerKeyMapping(rdbBatchOperation, key,
+                containerKeyMap.get(key));
+      } catch (IOException e) {
+        LOG.error("Unable to write Container Key Prefix data in Recon DB.",
+            e);
+      }
+    });
+
+    containerKeyCountMap.keySet().forEach((Long key) -> {
+      try {
+        reconContainerMetadataManager
+            .batchStoreContainerKeyCounts(rdbBatchOperation, key,
+                containerKeyCountMap.get(key));
+      } catch (IOException e) {
+        LOG.error("Unable to write Container Key Prefix data in Recon DB.",
+            e);
+      }
+    });
+
+    deletedContainerKeyList.forEach((ContainerKeyPrefix key) -> {
+      try {
+        reconContainerMetadataManager
+            .batchDeleteContainerMapping(rdbBatchOperation, key);
+      } catch (IOException e) {
+        LOG.error("Unable to write Container Key Prefix data in Recon DB.",
+            e);
+      }
+    });
+
+    reconContainerMetadataManager.commitBatchOperation(rdbBatchOperation);
+  }
+
   /**
    * Delete an OM Key from Container DB and update containerID -> no. of keys
    * count.
    *
    * @param key key String.
```

Comment:
change the javadoc accordingly to show that this is _preparing for batch deletion (IIUC)_. and list the new params. same for others.